semantic.json

```diff
@@ -8,66 +8,15 @@
       "themes": "src/themes/"
     },
     "output": {
-      "packaged": "out/",
-      "uncompressed": "out/components/",
-      "compressed": "out/components/",
-      "themes": "out/themes/"
+      "packaged": "dist/",
+      "uncompressed": "dist/components/",
+      "compressed": "dist/components/",
+      "themes": "dist/themes/"
     },
     "clean": "dist/"
   },
   "permission": false,
-  "autoInstall": false,
+  "autoInstall": true,
   "rtl": false,
-  "components": [
```

Comment:
Why are these components gone from the file now? Don't we use some of these?

Reply:
I think this is what was trying to accomplish with this PR. Is that accurate @noahpoz ?

Reply:
Yes. The way I originally configured it was cherry picking the desired components (which ended up being all of them anyway). That's what's going on here. Specifying the components like this isn't necessary when autoInstall is true.

Reply:
Sweet!